docs/development/dev_box_en.md

```diff
@@ -0,0 +1,47 @@
+# Dev Box
+
+This document contains the necessary information to start the development basebox locally with Vagrant.
+Instructions for building the image for the development basebox can be found [here](./building_basebox_en.md).
+
+### Configuration
+
+The configuration for the dev box is done via a `.vgarant.rb` file. This is read in from the `Vagrantfile` and can
+overwrite the configuration values from the `Vagrantfile`.
+The following configuration values can be specified (among others):
+
+| value                   | description                                    |
+|-------------------------|------------------------------------------------|
+| dogu_registry_url       | The URL of the dogu registry                   |
+| dogu_registry_username  | The username to login to the dogu registry     |
+| dogu_registry_password  | The password to login to the Dogu Registry     |
+| image_registry_url      | The URL of the image registry                  |
+| image_registry_username | The username to login to the image registry    |
+| image_registry_password | The password to login to the image registry    |
+| image_registry_email    | The e-mail address of the image registry user  |
+| helm_registry_url       | URL of the helmet registry                     |
+| helm_registry_username  | The username to login to the helmet registry   |
+| helm_registry_password  | The password to login to the helmet registry   |
```

Comment:
helmet registry - I love it :D -> helm registry

Reply:
Deepl :roll_eyes: